.circleci/config.yml

```diff
@@ -958,8 +958,21 @@ jobs:
             terraform destroy -auto-approve
       - run: *notify-slack-failure
 
+  // The noop job is a used as a very fast job in the verify-ci workflow because every workflow
+  // requires at least one job. It does nothing.
```

Comment:
```suggestion
  # The noop job is a used as a very fast job in the verify-ci workflow because every workflow
  # requires at least one job. It does nothing.
```